help/en/html/hardware/dccpp/index.shtml

```diff
@@ -50,20 +50,30 @@
 <h3>Command Stations</h3>
 
     Currently DCC++ Base Stations can be built from either the Arduino Uno
-    or Arduino Mega platforms.  JMRI should support either platform via USB.
+    or Arduino Mega platforms.  JMRI should support either platform via USB or Network connection.
 </div>
```

Comment:
This </div> should go - it's closing the 'mainContent' section too early causing the formatting to be a bit weird.